app/src/main/java/fi/sabriina/urbanhuikka/MainActivity.kt

```diff
@@ -223,7 +225,9 @@ class MainActivity : AppCompatActivity(), OnCardSwipeListener {
         CoroutineScope(Dispatchers.Main).launch {
             currentCard = gameStateViewModel.getNextCard(deck)
             if (currentCard != null) {
-                cardView.setCard(currentCard!!)
+                val assistingPlayer = gameStateViewModel.drawAssistingPlayer()
```

Comment:
Assisting player is now selected for all cards instead of just the ones including multiple players. This could be optimized by setting an identifier to cards that require multiple players.